test/test_models.py

```diff
@@ -36,18 +36,85 @@ def test_mfcc(self):
 class TestMelResNet(common_utils.TorchaudioTestCase):
 
     def test_waveform(self):
+        """
+        Create a tensor as the input of _MelResNet layer
+        and test if the output dimensions are correct.
+        """
```

Comment:
Please use the same docstring notation.

```
"""Short description on the first line as the opening of docstring

Then the more detailed explanation only if necessary.
Every test should be simple so if you need to add more explanation,
you should also consider the design of test and simplify it.
"""
```

Reply:
Updated.

Reply:
Thanks for also opening #751 to update this changes